components/Form/index.js

```diff
@@ -8,11 +8,21 @@ import { useRouter } from "next/router";
 export default function Form() {
   const router = useRouter();
   const [items, setItems] = useImmer(partList2);
+  const [additionalItems, setAdditionalItems] = useImmer([]);
```

Comment:
As far as I know, it's a kind of convention to name the setter function of the `useImmer` hook with `updateX` instead of `setX`?